jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-prototype.c

```diff
@@ -763,10 +763,7 @@ ecma_op_typedarray_set_with_typedarray (ecma_value_t this_arg, /**< this argumen
   /* 24.d, 25. srcByteIndex */
   ecma_length_t src_byte_index = 0;
 
-  if (src_arraybuffer_p != target_arraybuffer_p)
-  {
-    src_byte_index = src_byte_offset;
-  }
+  src_byte_index = src_byte_offset;
```

Comment:
IMO `src_byte_index` has became unnecessary, since `src_byte_offset` can be used directly.